cosmo-core/src/main/java/org/unitedinternet/cosmo/model/hibernate/HibBaseEventStamp.java

```diff
@@ -144,6 +145,10 @@ private static Calendar calendarFromString(String icaldata) {
     private static String calendarToString(Calendar value) {
         String calendar = null;
         try {
+            ValidationResult validationResult = value.validate(true);
+            if (validationResult.hasErrors()) {
+                throw new net.fortuna.ical4j.validate.ValidationException("calendar has validation errors");
```

Comment:
Here and below we throw `net.fortuna.ical4j.validate.ValidationException`, then we catch `jakarta.validation.ValidationException`.